api/api.go

```diff
@@ -63,7 +74,7 @@ func (response GQLResponseErrors) ToError() error {
 		messages = append(messages, response.Errors[i].Message)
 	}
 
-	return errors.New(strings.Join(messages, ": "))
+	return fmt.Errorf(strings.Join(messages, ": "))
```

Comment:
Why the change from the `errors` package to string-formatting (`fmt`) package here? IMO using the errors package is  more clear of the intent, and more clear still since there is no string-formatting going on.

Reply:
This may be due to the nature of how these 2 PRs were merged, sorry about that.

I will fix it and address the other comments tomorrow unless @johnswanson or @hannahhenderson get to it.

Reply:
The linter actually screamed at me for `errors.New` and demanded `fmt.Errorf`. I assumed that was because the error, in this case, is only a formatted string.

Reply:
@hannahhenderson If linter warnings failing builds becomes a nuisance, I think we can check for them in bash and choose to not fail the build:

https://github.com/alecthomas/gometalinter#exit-status